src/concepts/essentials/httpql.md

```diff
@@ -2,99 +2,167 @@
-- **ne**: No equal to `value`.
+- `eq`: **Equal to** the supplied value.
+- `gt`: **Greater than** the supplied value.
+- `gte`: **Greater than or equal to** the supplied value.
+- `lt`: **Less than** the supplied value.
+- `lte`: **Less than or equal to** the supplied value.
+- `ne`: **Not equal** to the supplied value.
 
 ### String/Bytes
 
-This category of Operators works on Fields that are text values (or bytes) like `path` and `raw`.
+This category of Operators works on Fields that are text or byte values (_such as `path` and `raw`_).
+
+- `cont`: **Contains** the supplied value.
+- `eq`: **Equal** to the supplied value.
+- `like`: The [SQLite LIKE Operator](https://www.sqlite.org/lang_expr.html#the_like_glob_regexp_match_and_extract_operators).
+- `ncont`: **Does not** contain the supplied value.
+- `ne`: **Not equal to** the supplied value.
+- `nlike`: The [SQLite NOT LIKE Operator](https://www.sqlite.org/lang_expr.html#the_like_glob_regexp_match_and_extract_operators).
 
-- **cont**: Contains `value` (case insensitive).
-- **eq**: Equal to `value`.
-- **like**: Sqlite `LIKE` [Operator](https://www.sqlite.org/lang_expr.html#the_like_glob_regexp_match_and_extract_operators). The symbol `%` matches zero or more characters (like `%.js` to match `.map.js`) and the symbol `_` matches one character (like `v_lue` to match `vAlue`).
-- **ncont**: Doesn't contain `value` (case insensitive).
-- **ne**: No equal to `value`.
-- **nlike**: SQLITE `NOT LIKE` Operator, see `like` for more details.
+::: tip TIPS
+
+- The `cont` and `ncont` Operators are case insensitive.
+- In SQLite - the `%` character matches zero or more characters (_such as `%.js` to match `.map.js`_) and the `_` character matches one character (_such as `v_lue` to match `vAlue`_).
+- The `like` Operator is case sensitive for unicode characters that are beyond the ASCII range.
+:::
 
 ### Regex
 
-This category of Operators works on Fields that are text values (or bytes) like `path` and `raw`.
+This category of Operators works on Fields that are text or byte values (_such as `path` and `raw`_).
+
+- `regex`: Matches the regex `/value.+/`.
+- `nregex`: Doesn't match the regex `/value.+/`.
+
+::: info
+Not all regex features are currently supported by Caido (_such as look-ahead expressions_) as they are not included in the regex library of Rust.
+:::
 
-- **regex**: Matches the regex `/value.+/`.
-- **nregex**: Doesn't match the regex `/value.+/`.
+::: tip
+Visit [https://regex101.com/](https://regex101.com/) and select **Rust** syntax for assistance in creating expressions.
+:::
 
-## 4. Value
+## Value
 
-This is the value against which the Field will be compared. The value is either an integer (like `1`), a string (`"value"`) or a regex (`/value/`) depending on the Field and Operator.
+This is the value against which the Field will be compared. The value type is either an **integer**, a **string** or a **regex** depending on the Field and Operator.
```

Comment:
I would rephrase to something along: `The base types are integer, string, regex and date/time depending on the Field and/or Operator`